client/src/pages/DetailsPage/item-details.module.scss

```
@@ -2,26 +2,179 @@
         }
 
+        .nameIndicator {
+            font-size: $fz-small-4;
+            color: $fc-5;
+            margin-bottom: 1.5%;
+        }
     }
 
-    .quantityIndicator {
-        background-color: green;
-        width: 20vw;
-        height: 10vw;
+    .ratingSection {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 5%;
+
+        img {
+            width: $fz-small-4;
+            margin-right: 1.4%;
+        }
+
+        .ratingIndicator {
+            font-size: $fz-small-4;
+            color: $fc-4;
+            margin-right: 1%;
+        }
+
+        .reviewIndicator {
+            font-size: $fz-small-4;
+            color: $fc-4;
+        }
     }
 
-    .quantitySelection {
-        font-family: $ff-roboto;
-        font-size: 5rem;
-        background-color: green;
+    hr {
+        color: #c0c0c0;
     }
 
+    .quantitySection {
+        margin-top: 4%;
+
+        .quantityTitle {
+            font-size: $fz-small-4;
+            color: $fc-4;
+            margin-bottom: 3%;
+        }
+
+        .selectQuantity {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin-bottom: 6%;
+
+            .minusQuantity,
+            .plusQuantity {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                font-size: $fz-large-1;
+                padding: 5.4%;
+                width: 2rem;
+                height: 2rem;
+                border-radius: 50%;
+                border-color: transparent;
+
+                &:hover {
+                    background-color: #DEFFE4;
+                    color: $fc-3;
+                    transform: scale(1.1);
+                    transition: transform 0.4s;
+                }
+
+                &:active {
+                    background-color: $bc-1;
+                    box-shadow: 0 0 5px #DEFFE4, 0 0 10px #DEFFE4, 0 0 20px #DEFFE4, 0 0 30px #DEFFE4;
+                    color: $fc-2;
+                    font-weight: 500;
+                }
+            }
+
+            .numberQuantity {
+                font-size: $fz-large-1;
+                margin: 0 16%;
+            }
+        }
+
+        .addCartButton {
+            font-size: $fz-small-5;
+            width: 80%;
+            height: 7vh;
+            margin-bottom: 7.5%;
+            background-color: $bc-2;
+            border-color: transparent;
+            border-radius: 10px;
+
+            &:hover {
+                background-color: #DEFFE4;
+                color: $fc-3;
+                transform: scale(1.01);
+                transition: transform 0.4s;
+                font-weight: 400;
+            }
+
+            &:active {
+                background-color: $bc-1;
+                box-shadow: 0 0 5px #DEFFE4, 0 0 10px #DEFFE4, 0 0 20px #DEFFE4, 0 0 30px #DEFFE4;
+                color: $fc-2;
+                font-weight: 600;
```

600 gibt es btw nicht nach 500 der nächst größere Schritt wäre 700.
```suggestion
                font-weight: 700;
```
```suggestion
                font-weight: $fw-bold;
```

---

habe die 700 erst mal übernommen. die variable habe ich noch nicht, lasse es deshalb erstmal offen

---

done!